.editorconfig

```diff
@@ -260,3 +260,16 @@ resharper_csharp_wrap_list_pattern = chop_if_long
 
 [src/Aquifer.Data/Migrations/*.cs]
 generated_code = true
+
+[*.{cs,vb}]
+dotnet_style_coalesce_expression = true:error
+dotnet_style_null_propagation = true:error
+dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
+dotnet_style_prefer_auto_properties = true
+dotnet_style_object_initializer = true:suggestion
+dotnet_style_collection_initializer = true
+dotnet_style_operator_placement_when_wrapping = beginning_of_line
+dotnet_style_prefer_collection_expression = when_types_loosely_match:suggestion
+tab_width = 4
+indent_size = 4
+end_of_line = crlf
```

Comment:
We had this in here at one point but it was causing issues cross-platform with git. Removing it lets the committed code live with `lf` which git likes, but then Windows machines will handle it during checkout as `crlf`. That's my understanding of it.

Reply:
PR: https://github.com/BiblioNexusStudio/aquifer-server/pull/415

Reply:
Also for reference looked at some MS repos to see what they were using for .editorconfig as well as their .gitattributes. Example: https://github.com/dotnet/efcore/blob/main/.editorconfig

Reply:
Thanks, I removed the `crlf` setting.  IMO it would be best to update everything to LF rather than have both LF and CRLF depending upon platform (but I'm not going to do that in this PR).  VS will respect this EOL setting (and I'm assuming Rider will as well).